Katana/Plastic/Value.swift

```diff
@@ -145,3 +187,65 @@ public struct Value: Equatable {
     return lhs.scalable == rhs.scalable && lhs.fixed == rhs.fixed
   }
 }
+
+/**
+ This allows `Value` instances to be initialized by integer and floating point literals,
+ having the scalable part equal to the numeric literal, and the fixed part equal to 0.
+ */
+extension Value: ExpressibleByIntegerLiteral, ExpressibleByFloatLiteral {
```

Comment:
Alright guys.. as you wish..
It still doesn't make any sense to me to create confusion everywhere just to handle a single case. I'm sure that it will lead to confusion in the future but of course I can't prove it.

```
but, saying that 200 is a .fixed/scalable(200) is a natural extension to me. At the end of the day you can create a Value starting from an Int using Value.fixed(200), this means that a Value and an Int can express the same thing.
```
1- A value is composed by two int (or double..). We should take a decision whether a single int is the same as a Value with fixed = 0 or scalable = 0. And then explain this to everyone 

2 - `Value.fixes(Int)` to me is like `Int(AString)`. I don't think anyone would automatically convert strings into int, right? They are different things, formed by different type of values. For the sake of simplicity during the layout phase we decided to allow the usage of numbers. It is cleaner and simpler to read. Ok fair enough (consider also that, in 99% of the cases, that number will be a scalable.. and here we also have a reason to pick one of the two). But this is a DSL for that specific case. Extend this DSL to everything, where things may be more complex, hard to understand and use cases change.. it is a BIG mistake to me